packages/framework/src/services/microservice/microservice-gateway.service.ts

```diff
@@ -45,6 +53,9 @@ class MicroserviceGatewayService {
     kafka: async (item) => {
       await KafkaGateway.publish(this.#gateways.kafka.get(item.broker), item);
     },
+    smb: (item) => {
+      SmbGateway.publish(this.#gateways.mqtt.get(item.broker), item);
```

Comment:
Think this has to be updated
```suggestion
      SmbGateway.publish(this.#gateways.smb.get(item.broker), item);
```